tasks/setup_env.yml

```diff
@@ -18,6 +18,14 @@
     - ca-certificates
   when: ansible_os_family in [ "Debian" ]
 
+- name: "[ENV] - Upgrade all packages to latest version."
+  ansible.builtin.apt:
+    upgrade: "yes"
+    force_apt_get: true
+  when:
+    - ansible_os_family in [ "Debian" ]
+    - (upgrade_packages_first is defined) and (upgrade_packages_first)
```

Comment:
If it is kept as suggest in the defaults, then yes, otherwise this version does the same. In any case, this is an error resilient way and that is why I have let it. Please advice how this shall be best handled.

Reply:
Example: A new OS is added. In this case, this code does not cause errors.

Reply:
Changed accordingly.

Reply:
I agree with @aalaesar, it can be simplified as mention in suggestion:
```suggestion
    - upgrade_packages_first
```

Reply:
Changed.